<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 0, 0, 0);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
   </property>
   <layout class="QGridLayout" name="centralLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="left_frame">
      <property name="minimumSize">
       <size>
        <width>720</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="left_frameLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="r_top_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="r_top_frameLayout">
          <item row="0" column="0">
           <widget class="QFrame" name="weather_frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="weather_frameLayout"/>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFrame" name="frame">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="l_bottom_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="l_bottom_frameLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="0" column="0" alignment="Qt::AlignBottom">
           <widget class="QPushButton" name="appBtn">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto Mono for Powerline</family>
              <pointsize>16</pointsize>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Apps</string>
            </property>
            <property name="icon">
             <iconset theme="system-run"/>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="l_middle_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="l_middle_frameLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="time_Label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>680</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>DSEG14 Classic</family>
              <pointsize>80</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel
{
color: rgba(255, 85, 0, 240);
}</string>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="right_frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="right_frameLayout" rowstretch="0,0,0">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="horizontalSpacing">
        <number>5</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="top_frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="top_frameLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QFrame" name="sensor_frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>135</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>135</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(42, 42, 62, 120);
border-radius: 20px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="sensor_frameLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="cpulabel">
               <property name="minimumSize">
                <size>
                 <width>39</width>
                 <height>39</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
               </property>
               <property name="text">
                <string>CPU:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QProgressBar" name="cpuBar">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QProgressBar
{
border: solid grey;
border-radius: 3px;
color: rgb(225, 225, 225);
background-color: rgba(0, 0, 0, 0);
}
QProgressBar::chunk 
{
background-color: rgba(255, 170, 127, 160);
border-radius :6px;
}  </string>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="ramLabel">
               <property name="minimumSize">
                <size>
                 <width>39</width>
                 <height>31</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
               </property>
               <property name="text">
                <string>RAM:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QProgressBar" name="ramBar">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QProgressBar
{
border: solid grey;
border-radius: 3px;
color: rgb(225, 225, 225);
background-color: rgba(0, 0, 0, 0);
}
QProgressBar::chunk 
{
background-color: rgba(134, 255, 148, 160);
border-radius :6px;
}  </string>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="cpu_tempLabel">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>31</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
               </property>
               <property name="text">
                <string>t  °C</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QProgressBar" name="tempBar">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QProgressBar
{
border: solid grey;
border-radius: 3px;
color: rgb(225, 225, 225);
background-color: rgba(0, 0, 0, 0);
}
QProgressBar::chunk 
{
background-color: rgba(85, 170, 255, 160);
border-radius :6px;
}  </string>
               </property>
               <property name="minimum">
                <number>20</number>
               </property>
               <property name="maximum">
                <number>80</number>
               </property>
               <property name="value">
                <number>19</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="format">
                <string>%p%°C</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QFrame" name="bootom_frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>135</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(42, 42, 62, 120);
border-radius: 20px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="bootom_frameLayout">
          <item>
           <widget class="QLabel" name="volume_label">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Fira Mono for Powerline</family>
              <pointsize>14</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgba(85, 170, 255, 200);
background-color: rgba(0, 0, 0, 0);</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="volume_dial">
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>170</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 85, 127);</string>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="notchesVisible">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="middle_frame">
         <property name="styleSheet">
          <string notr="true">
border-radius: 20px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="middle_frameLayout">
          <item row="1" column="0" colspan="2">
           <widget class="QFrame" name="net_frame">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>140</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="net_frameLayout">
             <item row="1" column="2">
              <widget class="QLabel" name="upLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(85, 255, 127);</string>
               </property>
               <property name="text">
                <string>0.0 b/s</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="dnLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgba(170, 170, 255, 255);</string>
               </property>
               <property name="text">
                <string>0.0 b/s</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgba(170, 170, 255, 255);</string>
               </property>
               <property name="text">
                <string>Download:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QLabel" name="interfaceLabel">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgba(255, 255, 255, 200);</string>
               </property>
               <property name="text">
                <string>Interface: eth0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(85, 255, 127);</string>
               </property>
               <property name="text">
                <string>Upload:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="3">
              <widget class="QLabel" name="ipLabel">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgba(255, 255, 255, 200);</string>
               </property>
               <property name="text">
                <string>IP: 0.0.0.0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
